src/comp/comp.py

```diff
@@ -21,51 +25,52 @@ def __repr__(self):
     Human("David", 31),
 ]
 
-# Write a list comprehension that creates a list of names of everyone
+# 1 Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == "D"]
```

Comment:
Thank you for making your code readable by writing out human instead of using something like 'h' instead.